### PR TITLE
custom-fields: document number and boolean widgets

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: docs/reference/widgets.md [hunk 1]
```diff
@@ -35,6 +35,71 @@ An input field for a single string value.
...
+
+- **description** `String` _required_: The description for the element's display. You should provide useful information on how the users should fill this field in.
+
+- **icon** `String` _optional_: The optional icon that your field might have. The value should be one of the [Semantic UI icons](https://react.semantic-ui.com/elements/icon/).
+
+- **required** `Boolean` _optional_: Define if the field should be required i.e display the `*` symbol.
+
+- **disabled** `Boolean` _optional_: Define if the field should be displayed as disabled thus no input can be filled.
+
+## BooleanInput
```
Comment (zzacharo) [inline docs/reference/widgets.md:71]: THis is a radio button and I just have internal guilts that I named it like that 😅 If you don't like it, I can rename and populate the change everywhere i.e demo site, react-invenio-forms... @ntarocco @ppanero

File: docs/reference/widgets.md [hunk 2]
```diff
@@ -35,6 +35,71 @@ An input field for a single string value.
...
+## NumberInput
+
+An input field for numbers e.g integers, float etc.
+
+### Usage
+
+```javascript
+<NumberInput
+  fieldPath="cern:experiment_id"
+  label="Experiment URL"
```
Comment (ppanero) [inline docs/reference/widgets.md:47]:
```suggestion
  label="Experiment Identifier"
```

**Review by ppanero — APPROVED** [review]
LGTM. I do agree that `BooleanCheckbox` is better, if you want to make the PRs but then they will be release before v10 (i.e. no need to make a chain of releases right now. Up to you ;)

**Comment by zzacharo** [comment]
I will do the changes and rename it!

**Pull request merged**